index.html

```diff
@@ -3363,6 +3363,14 @@ <h3>JSON-LD</h3>
 Model is available at <code>https://www.w3.org/2018/credentials</code>.
         </p>
 
+        <p>
+This specification restricts the usage of JSON-LD representations of 
+<a>credentials</a> and <a> presentations</a>. JSON-LD <a
+href="https://www.w3.org/TR/json-ld/#compacted-document-form">compact document
+form</a> MUST be utilized for all representations of the data model described by
```

Comment:
"all representations of the data model"... this seems not correct based on day 3 resolution.

we agreed that this only needs to be true for the "base media type" aka

`application/credential+ld+json` or `application/presentation+ld+json`

Reply:
> "all representations of the data model"... this seems not correct based on F2F day 3 resolution.

You're taking that phrase out of context.  The stricture is on "all representations of the data model **_described by this specification_**" which I believe we expect to be few, so your concern seems premature to me.

Reply:
> Resolution #1: The base media type for the VCDM is credential+ld+json. `@context` is required (MUST) in the base media type; other media types MAY choose to include `@context`. Serializations in other media types (defined by the VCWG) MUST be able to be transformed into the base media type. Another media type MUST identify if this transformation is one-directional or bi-directional. Bi-directional transformation MUST preserve `@context`. Transformation rules MUST be defined, but not necessarily by this WG..

https://www.w3.org/2017/vc/WG/Meetings/Minutes/2023-02-16-vcwg#resolution1

https://github.com/w3c/vc-data-model/issues/947#issuecomment-1434506542

I feel `credential+ld+json` uses "compact form" would be a clearer way of saying the same thing.

Seems the intention is aligned with resolution, and data integrity proofs can indeed use "non compact form" for VCs.

Reply:
I think we would do well by rewording the sentence —
```
JSON-LD <a
href="https://www.w3.org/TR/json-ld/#compacted-document-form">compact document
form</a> MUST be utilized for all representations of the data model described by
```
— to —
```
JSON-LD <a
href="https://www.w3.org/TR/json-ld/#compacted-document-form">compact document
form</a> MUST be utilized for all JSON-LD representations of the data model described by
```
— if that is indeed what was intended (which I don't think it was).

Given that we're defining the base media type, named `credential+ld+json` (or `vc+ld+json`), to which all other representations must be losslessly transformable, I think it would be better to say —
```
JSON-LD <a
href="https://www.w3.org/TR/json-ld/#compacted-document-form">compact document
form</a> MUST be utilized for all representations of the data model in the base 
media type, vc+ld+json (or credential+ld+json).
```

Reply:
Fixed in e63c0692c44b02f3d679250bcb3c2b48b7dd9e3d.